### PR TITLE
feat: Add German translations for Devise and Doorkeeper

### PR DESCRIPTION
**Pull request by hui00**
Converted the English content of both Devise and Doorkeeper configuration files into German. This adaptation is crucial for presenting the application interface in German for the target audience.

SVA-285

### CODE REVIEW

File: config/locales/doorkeeper.de.yml
Comment (donni106) [inline config/locales/doorkeeper.de.yml:1]: doorkeeper has locale files itself in a repo, for example https://github.com/doorkeeper-gem/doorkeeper-i18n/blob/main/rails/locales/de.yml

I found some English parts in there. But can you please compare some German parts, to have the best for us.
You could also think of making a PR for their translation file to have a better public German file for doorkeeper?

File: config/locales/devise.de.yml
Comment (donni106) [inline config/locales/devise.de.yml:1]: devise also has some translation files in their repository: https://github.com/tigrish/devise-i18n/blob/master/rails/locales/de.yml

Are there parts that are better or worse?

File: config/locales/doorkeeper.de.yml
Comment (hui00) [reply config/locales/doorkeeper.de.yml:1]: I've also noticed the same situation with Devise. I'm unsure whether we should opt for a complete translation or stick with what we already have in the doorkeeper.en.yml and devise.en.yml files? I can also create a PR for Doorkeeper

File: config/locales/devise.de.yml
Comment (hui00) [reply config/locales/devise.de.yml:1]: It would say 'equally

File: config/locales/doorkeeper.de.yml
Comment (donni106) [reply config/locales/doorkeeper.de.yml:1]: Let's stick with our version

**Review by donni106 — APPROVED** [review]

**Pull request merged**